tests/unit/test__fixture__helpers.py

```diff
@@ -0,0 +1,40 @@
+import os
+from unittest.mock import MagicMock
+
+from grasshopper.lib.fixtures import _get_tagged_scenarios
+
+
+def test_get_tagged_scenarios_happy():
+    config_mock = MagicMock()
+    config_mock.getoption = lambda a: "asdf"
+    raw_yaml_dict = {
+        "scenario1": {"tags": ["asdf"]},
+        "scenario2": {"tags": ["foo"]},
+    }
+    tagged_scenarios = _get_tagged_scenarios(raw_yaml_dict, config_mock, fspath="asdf")
+    assert tagged_scenarios == {"scenario1": {"tags": ["asdf", "scenario1"]}}
+
+
+def test_get_tagged_scenarios_happy_env_var():
+    config_mock = MagicMock()
+    config_mock.getoption = lambda a: None
+    os.environ["TAGS"] = "foo"
```

Comment:
would be better to use `patch.dict` to alter the contents of `os.environ` since that will act as a context manager and ensure that there aren't side effects caused by this test or side effects from other tests with this one. in fact, if for some reason this test ran before the first one, the first one in the file would fail.

you can apply a decorator like so
```
@patch.dict({"TAGS": "foo"}, clear=True)
def test_get_tagged_scenarios_happy_env_var():
...
```

Reply:
changed